service/src/main/resources/api/service_openapi.yaml

```diff
@@ -250,12 +250,9 @@ components:
           type: string
           description: Unique identifier of the Azure subscription
           format: uuid
-        resourceGroupName:
-          type: string
-          description: Name of the Azure resource group
-        applicationDeploymentName:
+        managedResourceGroupId:
           type: string
-          description: Name of the Azure application deployment
+          description: ID of the Azure managed resource group
```

Comment:
Does this expect an ID like `/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-terra-integration-test-20211118`? Maybe an example in the YAML description would help.

Reply:
No, this is the bare ID i.e,. mrg-terra-integration-test-20211118. I'll update the documentation.